### ISSUE TITLE
Voice Acting

### ISSUE THREAD

**Issue by eugeneloza** [davewx7/citadel#118]
Hi all!
(following discussion at Discord)
I can do the voice acting for the game (see example https://opengameart.org/content/voice-acting-sketch-open-dungeons ) this includes speech and some voice effects like groans/screams. However, I can't promise doing that quickly. My "best" part is funny characters, but seldom I can be serious too :) The variety of voices is large (even some non-human female voices are possible, like snake/naga in the example), my natural voice sounds like knight in the example.
In order to get started I need a kind of a LORE article on the creatures and the list of creatures. These may be undetailed, like "he is brave, but greedy, his voice is treacherous, he's not afraid of death but is highly disappointed by it", and a card picture if available.
Best if the lines (and maybe some reference, like some reference voice/sound, images, card rules) are also given, but if none there is no problem, I can do that myself.
So, to sum it up, I need a **list** of creatures with some **additional info** on their character/mood and anything else you can tell about the creature.

**Comment by galegosimpatico** [comment]
Related to https://github.com/davewx7/citadel/wiki/Voices-Acolyte and https://github.com/davewx7/citadel/wiki/Voices----Deephelm-Explorer.

**Comment by galegosimpatico** [comment]
Related to https://github.com/davewx7/citadel/wiki/Voices-Stoneclan-Berserker and https://github.com/davewx7/citadel/wiki/Voices:-Skeletal-Parasite.

**Comment by eugeneloza** [comment]
Happy new year!!!
And a small present (Berserker, Parasite and Inquisitor): https://drive.google.com/file/d/1v5YqqfGmeVLDia8itAcEEPG3Ee-YCEkC/view
Any ideas/suggestions are welcome :)
If it's ok, then I'll start processing the recording.

**Comment by galegosimpatico** [comment]
Such good new year present!

I find the first Berserker alternative too civilized and the first Parasite alternative too slow. It's nice there are couple alternatives for each character, and for each message, and out of script voices.

**Comment by davewx7** [comment]
Sorry for not responding to this sooner. Have been traveling. This sounds great! :) If you could process the recording and provide as individual wave files that would be great! Will love to have it in the game. :)
